CHANGELOG.md

```diff
@@ -8,6 +8,16 @@
 
 ### Fixed
 
+<a id="1_46_4"></a>
+
+## [1.46.4] - 2024-08-16
+
+### Changed
+- Update `@deltachat/stdio-rpc-server` and `deltachat/jsonrpc-client` to `1.142.6`
+  - Still try to create "INBOX.DeltaChat" if couldn't create "DeltaChat" #5870
```

Comment:
This matters only for some providers that require prefixes in folder names like "INBOX.". There was one more fix which seems more important:
fix: store_seen_flags_on_imap: Skip to next messages if couldn't select folder (https://github.com/deltachat/deltachat-core-rust/issues/5870)

Reply:
Yes, but mentioning both is also fine

Reply:
Hmm: when I look at https://github.com/deltachat/deltachat-core-rust/issues/5870 I see it is still open? So why is it mentioned in CHANGELOG? 
The issue worth to mention here would be "fix 'Couldn't select folder DeltaChat' log entry", since that's what desktop users may see
But the issue is not yet solved so I would not show both 

Anyway: this is just an additional information in the CHANGELOG so not worth to discuss in too much detail.
Users who are interested may look into the core CHANGELOG. So I added a direct link here

Reply:
It's still open because there's one more improvement PR still unmerged. But i agree that not mentioning it at all is fine, i think it didn't affect many users